packages/vsce/src/doc/meta/program.meta.ts

```diff
@@ -54,24 +54,33 @@ export const ProgramMeta: IResourceMeta<IProgram> = {
   },
 
   getHighlights(program: Resource<IProgram>) {
-    return [
-      {
+     const result: { key: string; value: any }[] = [];
+     result.push({
         key: l10n.t("Status"),
         value: program.attributes.status,
-      },
-      {
+      },)
+     if(program.attributes.language!="NOTDEFINED"){
```

Comment:
Some common styling for this file, and using a strict equals (`!==`) is preferred.

```suggestion
     if (program.attributes.language !== "NOTDEFINED"){
```